File: examples/user/resetPassword/node/resetPassword.js
const { UserService } = require("m3o/user");

// Reset password with the code sent by the "SendPasswordResetEmail" endoint.
async function resetPassword() {
  let userService = new UserService(process.env.MICRO_API_TOKEN);
  let rsp = await userService.resetPassword({
    code: "012345",
    confirmPassword: "NewPassword1",
    email: "joe@example.com",
    newPassword: "NewPassword1",
  });
  console.log(rsp);
}

resetPassword();

File: examples/user/resetPassword/go/resetPassword.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/user"
)

// Reset password with the code sent by the "SendPasswordResetEmail" endoint.
func ResetPassword() {
	userService := user.NewUserService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := userService.ResetPassword(&user.ResetPasswordRequest{
		Code:            "012345",
		ConfirmPassword: "NewPassword1",
		Email:           "joe@example.com",
		NewPassword:     "NewPassword1",
	})
	fmt.Println(rsp, err)
}
